app/src/main/res/values-en/strings.xml

```diff
@@ -835,30 +835,30 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_crossing_type_marked">Marked</string>
     <string name="quest_crossing_type_unmarked">Without road markings</string>
 
-    <string name="quest_cycleway_title2">"Is there a cycleway here? What type?"</string>
-    <string name="quest_cycleway_resurvey_title">"Is this still the cycleway situation here?"</string>
-    <string name="quest_cycleway_value_track">cycle track</string>
-    <string name="quest_cycleway_value_lane">cycle lane</string>
-    <string name="quest_cycleway_value_lane_soft">cycle lane (dashed markings)</string>
+    <string name="quest_cycleway_title2">"Is there a bike lane or path here? What type?"</string>
+    <string name="quest_cycleway_resurvey_title">"Is this still the cycling situation here?"</string>
+    <string name="quest_cycleway_value_track">separated bike path</string>
```

Comment:
Is this for what Americans call “protected bike lanes”, or for dedicated bike paths paralleling the road? In any case, “separate” would be more grammatical than “separated” here.

Reply:
I know I could certainly use some clarification on what's intended.  I just guessed from the original text.  There are bike paths like the King Street bike path in Honolulu where I'm not sure what kind this would be since it's in the road but separated by bollards (for example).

Reply:
A cycle track is a bike lane that is off the street, i.e. behind the curb, next to the sidewalk.

<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/82/Ciclovia_Povoa_Varzim.jpg/639px-Ciclovia_Povoa_Varzim.jpg"/>

<img src="https://upload.wikimedia.org/wikipedia/commons/8/86/Vrijliggend_fietspad.JPG" />

Not everyone may agree, but I think `cycleway=track` is also tagged on bike paths that are protected from the rest of the street by other means than a curb, like this one here:

<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/eb/Green_cycle_track.jpg/569px-Green_cycle_track.jpg"/>

Reply:
so basically the bicycle equivalent of `footway=sidewalk`?

Reply:
No, it's still tagged on the road-way. If the cycleway is mapped as a separate way, on the road, `cycleway=separate` is tagged.

Reply:
I’m not sure American English has a natural word encompassing the examples above that everyone would interpret reliably. [NACTO is trying to popularize the term “cycle track”](https://nacto.org/publication/urban-bikeway-design-guide/cycle-tracks/), but that doesn’t necessarily mean anyone outside the field of traffic engineering has ever heard of it. Their definition would exclude the example with the grassy verge above. I think many people would refer to the first two examples imprecisely as “[sidepath](https://ruraldesignguide.com/physically-separated/sidepath)”, but the second example can also be called a “[separated bike lane](https://ruraldesignguide.com/physically-separated/separated-bike-lane)”. The third example is unequivocally a protected bike lane.

“Sidepath or protected bike lane”?

Reply:
Okay... so what wording to use now...?

Note that an illustration is shown alongside the title
<img width=128 src="https://raw.githubusercontent.com/streetcomplete/StreetComplete/master/res/graphics/cycleway/track.svg">

Reply:
That would be a brick-paved bike lane. 😅

As far as I can tell, “Sidepath or protected bike lane” would be the most succinct way to describe `cycleway=track`, but I’m probably missing some nuance in that tag’s technical definition. See the illustrations for Class I and Class IV in [this Caltrans guide](http://lvbikecoalition.org/wp-content/uploads/2017/12/caltrans-d4-bike-plan_bikeway-classification-brochure_072517.pdf#page=2).

Reply:
> That would be a brick-paved bike lane. 😅

Yeah but don't you see the curb? Eh, maybe it's only well noticeable when compared to the picture for cycleway lane. Anyway, I will use your suggestion: 
```suggestion
    <string name="quest_cycleway_value_track">sidepath or protected bike lane</string>
```